etc-conf/rhn-migrate-classic-to-rhsm.completion.sh

```diff
@@ -24,4 +24,4 @@ _rhn-migrate-classic-to-rhsm()
 	return 0
 }
 
-complete -F _rhn-migrate-classic-to-rhsm rhn-migrate-classic-to-rhsm
+complete -F _rhn-migrate_classic_to_rhsm rhn-migrate-classic-to-rhsm
```

Comment:
"_rhn-migrate_classic_to_rhsm" shouldn't it be an underscore between rhn and migrate?